src/core/server/config/mocks.ts

```diff
@@ -6,33 +6,10 @@
  * Side Public License, v 1.
  */
 
-import type {
-  getEnvOptions as getEnvOptionsTyped,
-  rawConfigServiceMock as rawConfigServiceMockTyped,
-  configServiceMock as configServiceMockTyped,
```

Comment:
Cleaning the imports/re-exports due to the new package (those will be removed in a follow-up PR to directly import from the mock package from within Core)